e2e/process-services-cloud/apps-section-cloud.e2e.ts

```diff
@@ -16,10 +16,11 @@
  */
 
 import { browser } from 'protractor';
-import { LoginSSOPage, SettingsPage } from '@alfresco/adf-testing';
+import { ApiService, GroupIdentityService, IdentityService, LoginSSOPage, RolesService, SettingsPage } from '@alfresco/adf-testing';
 import { AppListCloudPage } from '@alfresco/adf-testing';
 import { NavigationBarPage } from '../pages/adf/navigationBarPage';
 import resources = require('../util/resources');
+import CONSTANTS = require('../util/constants');
```

Comment:
The constants should be part of identity service class.
And you should use it like
`IdentityService.roles.aps_user`